### PR TITLE
Add a guide for setting up Access Graph on-prem

### PR DESCRIPTION
**Pull request by justinas**
This PR adds a basic guide for setting up the Teleport Access Graph service in a self-hosted environment.

Preview: https://docs-l482ha40g-goteleport.vercel.app/docs/access-graph/getting-started-on-prem/

### CODE REVIEW

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
```
Comment (justinas) [inline docs/pages/access-graph/getting-started-on-prem.mdx:8]: Wanted to add this call-out, since this "Getting started" guide is the only one we have for now (there's no equivalent for Team/Cloud - but there's nothing to set up manually on Team either).

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+* A running Teleport Enterprise cluster v15.0.0 or later.
+* Docker version v(=docker.version=) or later.
+* A PostgresSQL database server v14 or later.
```
Comment (justinas) [inline docs/pages/access-graph/getting-started-on-prem.mdx:18]: I validated this by running integration tests with containerized Postgres v14. https://github.com/gravitational/access-graph/pull/429 introduces a Postgres 14+ feature (recursive CTEs), so in a very near future we will not be supporting 13 and earlier.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
```
Comment (justinas) [inline docs/pages/access-graph/getting-started-on-prem.mdx:16]: Self-hosted TAG should also work with the next v14 version (v14.3.4). Will update when such version is out.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+- Docker version v(=docker.version=) or later.
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:17]: Technically, docker is not required. Any container runtime will work.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+- A PostgresSQL database server v14 or later.
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:18]: I'd add that RDS is also supported.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+* A running Teleport Enterprise cluster v15.0.0 or later.
+* Docker version v(=docker.version=) or later.
+* A PostgresSQL database server v14 or later.
+  * Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to Postgres with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+* A TLS certificate for the Access Graph service
+* The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
+
```
[the remark this answers is not in the document]
Comment (jakule) [reply docs/pages/access-graph/getting-started-on-prem.mdx:24]: @klizhentas Good question. Ideas?

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 4 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:17]: That's true. However, our other guides that use containerized images seem to point towards Docker in a similar way, e.g. https://goteleport.com/docs/management/export-audit-events/fluentd/#prerequisites

Not sure how we'd change up instructions to `docker run`. Maybe somehow say "Docker or other container runtime, but we provide instructions for Docker here" 🙂

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 3 above)
Comment (tigrato) [reply docs/pages/access-graph/getting-started-on-prem.mdx:16]: It's out now

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
+- Docker version v(=docker.version=) or later.
+- A PostgresSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
```
Comment (tigrato) [inline docs/pages/access-graph/getting-started-on-prem.mdx:19]:
```suggestion
  - Access Graph needs a dedicated PostgresSQL compatible database to store its data.
```

should we refer as postgresql compatible database?

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+    The user that TAG connects to Postgres with needs to be an owner of this database, or have similar broad permissions:
```
Comment (tigrato) [inline docs/pages/access-graph/getting-started-on-prem.mdx:20]:
```suggestion
    The user that TAG connects to database with needs to be an owner of this database, or have similar broad permissions:
```

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+- A TLS certificate for the Access Graph service
+- The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
+
+</TabItem>
+</Tabs>
+
+## Step 1/3. Set up the Teleport Access Graph service
+
+You'll need a copy of your Teleport cluster's host certificate authority (CA) on the machine that hosts the Access Graph service.
```
Comment (tigrato) [inline docs/pages/access-graph/getting-started-on-prem.mdx:30]: Can we give a bit of insight why this is necessary? 
Refer that Auth and Proxy will use certificates signed by HostCA to dial to...

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 3 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:16]: Not yet AFAIK? The latest is 14.3.3, pre self-hosted support. https://github.com/gravitational/teleport/releases/tag/v14.3.3

Comment (tigrato) [reply docs/pages/access-graph/getting-started-on-prem.mdx:16]: It's cut and you can download it from https://goteleport.com/download/
Roman will create the release in github

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 5 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:18]: Good point. We also have IAM auth support, I can try to document it (+ validate since it has sat unused for a bit).

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 7 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:19]: I'm not so sure we should commit to "PostgreSQL-compatible", since there is no one definition of what that means. E.g. CockroachDB claims to be "compatible", but we have already found that TAG does not currently run on Cockroach.

Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:19]: Btw the list sub-item talks about a "database" as an object inside of Postgres - one Postgres server/cluster can host many "databases", not a database as in an instance.

Comment (jakule) [reply docs/pages/access-graph/getting-started-on-prem.mdx:19]: I'd be careful as Cockarouch is Postgres compatible, but TAG won't work with it. I also don't know what other backends we will include with this phrase.

Comment (tigrato) [reply docs/pages/access-graph/getting-started-on-prem.mdx:19]: I mean, not even cockroachdb say they are compatible with postgres 
> CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. This means that existing applications built on PostgreSQL can often be migrated to CockroachDB without changing application code.

They implement the wire protocol and some functionalities but they do not claim to be fully compatible

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (klizhentas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:24]: just add a note section

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v14.3.4 or later.
```
Comment (benarent) [inline docs/pages/access-graph/getting-started-on-prem.mdx:16]:
```suggestion
- A running Teleport Enterprise cluster v14.3.4 or later.
- An updated `license.pem` with Teleport Access Graph enabled. 
```

Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:16]: Thanks, I added this to the list (as "Teleport Policy", as that's the product we sell), and removed the standalone notice box.

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 4 above)
Comment (jakule) [reply docs/pages/access-graph/getting-started-on-prem.mdx:18]: Let's leave Docker for now. I'd assume that most people understand how to run a container.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+    #   aws_region: us-west-2
+
+# IP address (optional) and port for the TAG service to listen to.
+# This is the default value. This key can be omitted to listen on port 50051 on all interfaces.
+address: ":50051"
+
+tls:
+  # File paths of PEM-encoded TLS certificate and private key for the TAG server.
+  cert: /etc/access_graph/tls.crt
+  key: /etc/access_graph/tls.key
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:78]: This guide doesn't mention anything about how those certificates should be generated. Here is my proposal https://github.com/gravitational/access-graph/issues/445

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+
+Then, create a configuration file for the TAG service, similar to this:
+
+```yaml
+backend:
+  postgres:
+    # This uses the PostgreSQL connection URI format, see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS
+    # A stricter `sslmode` value is strongly recommended,
+    # e.g. `sslmode=verify-full&sslrootcert=/etc/access_graph/my_postgres_ca.crt`.
+    # For a full reference on possible parameters see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:64]: nit: Do you know if our connection string is parsed in any way by `pgx`? If yes, then the supported args could be different.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+In the YAML config for the Auth service, add a new top-level section for Access Graph configuration.
+
+```yaml
+access_graph:
+  enabled: true
+  # host:port where the TAG service is listening
+  endpoint: access-graph.example.com:50051
+  # Optionally, specify a trusted CA we expect the TAG service certificate to be signed by.
+  # If not specified, the system trust store will be used.
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:102]: I'd rephrase it. Starting with `optionally` suggests that you can skip this step, which is only true when the TAG's CA is added to a system store.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+## Step 3/3. View the Access Graph in the Web UI
+
+You can find Access Graph in the "Access Management" tab in the Web UI.
+![Access Management menu item](../../img/access-graph/menu-item.png)
+
+To access the interface, your user must have a role that allows `list` and `read` verbs on the `access_graph` resource, e.g.:
+
+```yaml
+kind: role
+version: v5
```
Comment (jakule) [inline docs/pages/access-graph/getting-started-on-prem.mdx:117]: nit: Do you use v6 or v7 right now by default?

**Review by jakule — APPROVED** [review]
I added a few comments, but overall, LGTM.

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 12 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx:64]: `pgx` would have to parse it as they don't use `libpq` AFAIK. However, they link to the same documentation, so I'd assume they keep 100% (or close to it) parity https://pkg.go.dev/github.com/jackc/pgx/v5@v5.5.3/pgconn#ParseConfig

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 14 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx]: Updated to v6, since that is the default on v14 and v15.

File: docs/img/access-graph/menu-item.png
Comment (zmb3) [inline docs/img/access-graph/menu-item.png:1]: Should we at least highlight the access graph menu item here? I had to scan the screenshot for a while before I found it.

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/access-graph/getting-started-on-prem.mdx:8]: This almost reads as if the first step to a "getting started on prem" guide is to sign up for a cloud trial.

Can we clarify by saying something like:

"Teleport Access Graph is enabled by default for Teleport Team users. You can skip the setup process here and start using Access Graph right away by signing up for a Teleport Team trial."

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+
+- A running Teleport Enterprise cluster v14.3.4 or later.
+- An updated `license.pem` with Teleport Policy enabled.
+- Docker version v(=docker.version=) or later.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to the database with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+  - Amazon RDS for PostgreSQL is supported.
+- A TLS certificate for the Access Graph service
```
Comment (zmb3) [inline docs/pages/access-graph/getting-started-on-prem.mdx:24]: If this is not discussed below, we should clarify the cert requirements (what domains, for example)

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+  allow:
+    rules:
+    - resources:
+      - access_graph
+      verbs:
+      - list
+      - read
+```
+
+The automatically provisioned `editor` role has the required permissions by default.
```
Comment (zmb3) [inline docs/pages/access-graph/getting-started-on-prem.mdx:130]:
```suggestion
The preset `editor` role has the required permissions by default.
```

File: docs/img/access-graph/menu-item.png
Comment (justinas) [reply docs/img/access-graph/menu-item.png:1]: Good point, PTAL at the change in f346f3cc14526ecd89e5724b78884357e5acbee9 , it should be a little better.

File: docs/config.json [hunk 17]
```diff
@@ -476,7 +476,19 @@
...
           "slug": "/access-graph/",
-          "forScopes": ["team"]
+          "forScopes": [
+            "team",
+            "enterprise"
+          ],
+          "entries": [
+            {
+              "title": "Getting Started On-Premises",
+              "slug": "/access-graph/getting-started-on-prem/",
```
Comment (ptgott) [inline docs/config.json:486]: It's a little counterintuitive to me that the `/access-graph` path is part of "Manage Access", but isn't part of the `/access-controls` path.

File: docs/config.json [hunk 18]
```diff
@@ -476,7 +476,19 @@
...
           "title": "Access Graph",
           "slug": "/access-graph/",
-          "forScopes": ["team"]
+          "forScopes": [
+            "team",
+            "enterprise"
+          ],
+          "entries": [
+            {
+              "title": "Getting Started On-Premises",
```
Comment (ptgott) [inline docs/config.json:485]:
```suggestion
              "title": "Self-Hosting TAG",
```
Since the instructions can also apply to cloud hosted compute infrastructure, unless I'm missing something?

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 19]
```diff
@@ -0,0 +1,133 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:2]:
```suggestion
title: Run Teleport Access Graph on Self-Hosted Clusters
```
For precision

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 20]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+<Notice type="note">
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:5]: No need for tags here. I would also add a paragraph here that summarizes the scope of this page (what would a user hope to learn by following it?).

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 21]
```diff
@@ -0,0 +1,133 @@
...
+Teleport Access Graph is enabled by default for Teleport Team users.
+You can skip the setup process here and start using Access Graph right away
+by signing up for a [Teleport Team trial](https://goteleport.com/signup/).
+</Notice>
+
+## Prerequisites
+
+<Tabs>
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:13]: No need for Tabs if there's a single TabItem.

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 22]
```diff
@@ -0,0 +1,133 @@
...
+- Docker version v(=docker.version=) or later.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to the database with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+  - Amazon RDS for PostgreSQL is supported.
+- A TLS certificate for the Access Graph service
+  - The TLS certificate must be issued for "server authentication" key usage,
+    and must list the IP or DNS name of the TAG service in an X.509 v3 `subjectAltName` extension.
+- The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:28]: This is the first mention of the architecture of the setup. Since architecture often trips up users, we should add a brief description of the components involved. (I'd recommend doing this before the Prerequisites.)

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 23]
```diff
@@ -0,0 +1,133 @@
...
+
+</TabItem>
+</Tabs>
+
+## Step 1/3. Set up the Teleport Access Graph service
+
+You'll need a copy of your Teleport cluster's host certificate authority (CA) on the machine that hosts the Access Graph service.
+TAG requires incoming connections to be authenticated via host certificates that the host CA issues to Auth and Proxy services.
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:36]:
```suggestion
TAG requires incoming connections to be authenticated via host certificates that the host CA issues to the Auth Service and Proxy Service.
```
Since these are proper names in the context of Teleport

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 24]
```diff
@@ -0,0 +1,133 @@
...
+
+The host CA can be retrieved and saved into a file in one of the following ways:
+
+<Tabs>
+<TabItem label="Via curl">
+```console
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:42]:
```suggestion
```code
```
For consistency with other docs pages

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 25]
```diff
@@ -0,0 +1,133 @@
...
+$ mkdir /etc/access_graph
+$ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=tls-host' > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+
+<TabItem label="Via tctl">
+```console
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:49]:
```suggestion
```code
```

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 26]
```diff
@@ -0,0 +1,133 @@
...
+$ mkdir /etc/access_graph
+$ tsh login --proxy=<Var name="teleport.example.com" />
+$ tctl get cert_authorities --format=json \
+    | jq -r '.[] | select(.spec.type == "host") | .spec.active_keys.tls[].cert' \
+    | base64 -d > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+</Tabs>
+
+Then, create a configuration file for the TAG service, similar to this:
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:59]: We should make it more explicit that this configuration file should be on the server running the TAG service

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 27]
```diff
@@ -0,0 +1,133 @@
...
+  - /etc/access_graph/teleport_host_ca.pem # A full path to the file containing the Teleport cluster's host CA certificate.
+```
+
+Finally, start the TAG service using Docker as follows:
+
+```console
+$ docker run -v <path-to-config>:/app/config.yaml -v /etc/access_graph:/etc/access_graph public.ecr.aws/gravitational/access-graph:(=access_graph.version=)
+```
+
+## Step 2/3. Update Teleport Auth server configuration
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:95]:
```suggestion
## Step 2/3. Update the Teleport Auth Service configuration
```
For consistency in product names

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 28]
```diff
@@ -0,0 +1,133 @@
...
+
+In the YAML config for the Auth service, add a new top-level section for Access Graph configuration.
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:97]:
```suggestion
In the YAML config for the Auth Service, add a new top-level section for Access Graph configuration.
```
Product name consistency

File: docs/pages/access-graph/getting-started-on-prem.mdx [hunk 29]
```diff
@@ -0,0 +1,133 @@
...
+access_graph:
+  enabled: true
+  # host:port where the TAG service is listening
+  endpoint: access-graph.example.com:50051
+  # Specify a trusted CA we expect the TAG server certificate to be signed by.
+  # If not specified, the system trust store will be used.
+  ca: /etc/access_graph_ca.pem
+```
+
+Then, restart Auth servers, followed by Proxy servers.
```
Comment (ptgott) [inline docs/pages/access-graph/getting-started-on-prem.mdx:109]:
```suggestion
Then, restart Auth Service instances, followed by Proxy Service instances.
```
Product name consistency

File: docs/config.json (hunk 18 above)
Comment (justinas) [reply docs/config.json:485]: That's true, this tries to communicate using TAG with a self-hosted cluster (vs Teleport Cloud).

I liked "Getting Started" since we use that for a bunch of other features and it is an obvious first place to click.

"Self-Hosting TAG" perhaps fails to communicate upfront that this is required when running against a self-hosted Teleport cluster. I wonder if there's a way to phrase the entry so it roughly matches the full page title:

> Run Teleport Access Graph on Self-Hosted Clusters

while being a bit shorter (which we probably want to avoid clunky menu items)

File: docs/config.json (hunk 17 above)
Comment (justinas) [reply docs/config.json]: Moved under `/access-controls/` in 35e9efd14b

File: docs/pages/access-graph/getting-started-on-prem.mdx (hunk 26 above)
Comment (justinas) [reply docs/pages/access-graph/getting-started-on-prem.mdx]: We do mention it at the start of this section https://github.com/gravitational/teleport/pull/37661/files/e9b522fce41ba99443d5151c2e8ff9482b124336#diff-351d93272d6e47e36224548eec736412a333de6d70c937e1512ccfaf61faa8f0R35 , but I'll re-iterate here.

**Review by ptgott — APPROVED** [review]
